extension/test/data_test.ts

```diff
@@ -69,4 +69,60 @@ describe('data.ts', function () {
       ).to.include.something.like({ entry: /^ＤＶＤ-ＲＯＭ .*/ });
     });
   });
+
+  describe('makeText', function () {
+    it('should return the text for a word dictionary entry', function () {
+      const entry = rcxDict.wordSearch('あ', /* doNames= */ false);
+      if (!entry) {
+        throw new Error(
+          `rcxDict.wordSearch('あ', /* doNames= */ false) returns null`
+        );
+      }
+      expect(rcxDict.makeText(entry, /* max= */ 1)).to.equal(
+        'あ\t(int) (1) ah; oh; (int) (2) hey!\n',
+        `word: 'あ', doNames: false, max: 1`
+      );
+      expect(rcxDict.makeText(entry, /* max= */ 2)).to.equal(
```

Comment:
Prefer to test a single behavior per test. In this case, you're testing two different `max` values ensuring it works.

Instead, I would make this second assert its own  `it` block with a name including something like "includes entries up to max" or similar.

You can share the entry by using a second level `describe` which scopes tests that are about word search.

Reply:
Good idea! I'll do that.